extensions/src/ACRE2Shared/_CONSTANTS.h

```diff
@@ -15,7 +15,8 @@
 #undef _DEBUG_DIALOG
 
 
-#define PIPE_TIMEOUT 10000
+//#define PIPE_TIMEOUT 10000
+#define PIPE_TIMEOUT 5000 //Lowered this as a 10 second wait before being moved is very noticable. This is however my opinion/preference, so is not required.
```

Comment:
Should be okay - just lowers the timeout from the TeamSpeak 3 plugin waiting for a message from the game.